SensusService/Protocol.cs

```diff
@@ -376,6 +377,10 @@ public static void RunUnitTestingProtocol(Stream unitTestingProtocolFile)
         private string _name;
         private List<Probe> _probes;
         private bool _running;
+        private bool _scheduledToStart;
+        private bool _scheduledToStop;
+        private string _scheduledStartCallbackId;
+        private string _scheduledStopCallbackId;
```

Comment:
Can we eliminate the two bools above and use null callback IDs to indicate that the protocol is not scheduled to start/stop?